$OpenBSD: patch-tools_generate_prefs_xsl,v 1.1 2013/06/18 18:45:49 sthen Exp $

/usr/obj/ports/darktable-1.2.1/build-amd64/src/preferences_gen.h:1352: warning: format '%lld' expects type 'long long int', but argument 4 has type 'long int'

--- tools/generate_prefs.xsl.orig	Tue Jun 18 19:30:13 2013
+++ tools/generate_prefs.xsl	Tue Jun 18 19:30:28 2013
@@ -218,7 +218,7 @@
     gtk_spin_button_set_value(GTK_SPIN_BUTTON(widget), dt_conf_get_int64("</xsl:text><xsl:value-of select="name"/><xsl:text>") * factor);
     g_signal_connect(G_OBJECT(widget), "value-changed", G_CALLBACK(preferences_callback_</xsl:text><xsl:value-of select="generate-id(.)"/><xsl:text>), NULL);
     g_signal_connect(G_OBJECT(dialog), "response", G_CALLBACK(preferences_response_callback_</xsl:text><xsl:value-of select="generate-id(.)"/><xsl:text>), widget);
-    snprintf(tooltip, 1024, _("double click to reset to `%"PRId64"'"), (gint64)(</xsl:text><xsl:value-of select="default"/><xsl:text> * factor));
+    snprintf(tooltip, 1024, _("double click to reset to `%"PRId64"'"), (int64_t)(</xsl:text><xsl:value-of select="default"/><xsl:text> * factor));
     gtk_object_set(GTK_OBJECT(labelev),  "tooltip-text", tooltip, (char *)NULL);
 </xsl:text>
 	</xsl:template>
